dompurify/dompurify.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for DOM Purify
+// Project: https://github.com/cure53/DOMPurify
+// Definitions by: Dave Taylor <http://davetayls.me>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+interface IDOMPurify {
+  sanitize(s:string):string;
+  addHook(hook:string, cb:(currentNode:Element, data:any, config:any) => Element):void;
+}
+
+declare var DOMPurify:IDOMPurify;
```

Comment:
please move this line into `declare module`. see https://github.com/borisyankov/DefinitelyTyped/blob/master/gulp/gulp.d.ts#L281-L284